utils/registry/error.go

```diff
@@ -0,0 +1,51 @@
+package registry
+
+import (
+	"fmt"
+	"github.com/layer5io/meshkit/errors"
+)
+
+var (
+	ErrAppendToSheetCode      = "mesheryctl-1116"
```

Comment:
Update the error code, please. It's meshkit.

To reduce manual work, replace the code with empty string and run meshkit error utility.